packages/remix-dev/cli/migrate/resolveInput.ts

```diff
@@ -0,0 +1,39 @@
+import inquirer from "inquirer";
+
+import { migrations } from "./migrations";
+
+const resolveProjectDir = (input?: string): string => {
+  return input || process.env.REMIX_ROOT || process.cwd();
+};
+
+const resolveMigrationId = async (input?: string): Promise<string> => {
+  let { migrationId } = await inquirer.prompt<{ migrationId: string }>([
+    {
+      name: "migrationId",
+      message: "Which migration would you like to apply?",
+      type: "list",
+      when: !input,
+      pageSize: migrations.length,
+      choices: migrations.map((m) => ({
+        name: `${m.id}: ${m.description}`,
+        value: m.id,
+      })),
```

Comment:
What about loosing the unnecessary intermediate variable?

```suggestion
      choices: migrations.map(({ description, id }) => ({
        name: `${id}: ${description}`,
        value: id,
      })),
```